server/README.md

```diff
@@ -0,0 +1,131 @@
+# Asherah Server
+Asherah Server is intended to be used by those who wish to utilize application-layer encryption but are unable to take advantage of the SDK directly, e.g., Asherah lacks an implementation in their preferred programming language.
+
+Table of Contents
+=================
+
+  * [Overview](#overview)
+  * [Samples](#samples)
+    * [Docker Compose](#docker-compose)
+    * [Kubernetes](#kubernetes-kind)
+  * [Server Development](#server-development)
+
+## Overview
+Asherah Server is as a light-weight service layer built atop the Asherah SDK with encrypt/decrypt functionality exposed via a [gRPC](https://grpc.io) service. It uses a Unix domain socket for local inter-process communication and is designed to be deployed along side your application.
+
+To integrate with the service you will need to generate client interfaces from the [.proto](./protos/appencryption.proto) service definition. Detailed instructions for generating client code in a number of languages can be found in the [gRPC Quick Starts](https://grpc.io/docs/quickstart/).
+
+## Samples
+Full code for the client implementations used in the following samples, as well as the multi-container application configurations, can be found in the [samples](./samples) directory.
+
+### Docker Compose
+In this section we'll use Docker Compose to launch a multi-container application comprised of a simple Python client and an Asherah Server sidecar.
+
+#### Prerequisites
+Ensure Docker Compose is installed on your local system. For installation instructions, see [Install Docker Compose](https://docs.docker.com/compose/install/).
+
+#### Build and run the sample application
+From the [samples](./samples) directory run `docker-compose up` which will launch the application defined in [docker-compose.yaml](./samples/docker-compose.yaml).
```

Comment:
should this part be migrated to the README in samples directory?
As we are showing the `pwd` as `samples` in the below console snippet.

Reply:
There is no README in the samples directory and I think we're fine including this here for the time being. We can revisit if the scope of this doc balloons.